### DISCUSSION TITLE
Добавить карты других кампусов

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [0niel/university-app#222] [Идеи]
https://www.mirea.ru/contacts/address/

**Comment by 0niel** [comment]
На счёт карты вроде бы смогли договориться с администрацией вуза. Ждём, когда нам предоставят хоть какие-нибудь схемы. У нас нет возможности ездить по кампусам и самим составлять карты =(

**Comment by Dragonprod** [comment]
Карты доступны на https://map.mirea.ru/
